### PR TITLE
Create CODE_OF_CONDUCT.md

### PR DESCRIPTION
**Pull request by jgrund**

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+## Scope
+
+This Code of Conduct applies both within project spaces and in public spaces when an individual is representing the project or its community. Examples of representing a project or community include using an official project e-mail address, posting via an official social media account, or acting as an appointed representative at an online or offline event. Representation of a project may be further defined and clarified by project maintainers.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be reported by contacting the project team at joe.grund@intel.com. The project team will review and investigate all complaints, and will respond in a way that it deems appropriate to the circumstances. The project team is obligated to maintain confidentiality with regard to the reporter of an incident. Further details of specific enforcement policies may be posted separately.
```
Comment (brianjmurrell) [inline CODE_OF_CONDUCT.md]: Should we use some kind of _role address_ or alias here, and in general for all general types of contact?

Comment (jgrund) [inline CODE_OF_CONDUCT.md]: I'd be fine with that, we just need to create one.

Comment (brianjmurrell) [inline CODE_OF_CONDUCT.md]: That should probably be a DCO ticket.  We might already have an alias that is appropriate.  Something that goes to the whole team perhaps.

Or maybe there is a preference somewhere that it go only to you.  In any case, what we present should be consistent and we do actual delivery targeting behind the scenes.

Comment (jgrund) [inline CODE_OF_CONDUCT.md]: Any thoughts on naming?

Comment (jgrund) [inline CODE_OF_CONDUCT.md]: Actually, nm. There is some conventions in place for naming.

**Comment by jgrund** [comment]
Going to land as-is and update before 4.0. Tracking this issue: https://github.com/intel-hpdd/intel-manager-for-lustre/issues/157

**Pull request merged**